doc/api/path.md

```diff
@@ -20,7 +20,7 @@ paths are being used.
 
 For example, using the `path.basename()` function with the Windows file path
 `C:\temp\myfile.html`, will yield different results when running on POSIX than
-when run on Windows:
+when running on Windows:
```

Comment:
While we're in here, how about we take this opportunity to simplify the text? It's needlessly complex. It can probably be joined with the preceding paragraph too. How about just this?:

```markdown
So using `path.basename()` might yield different results on POSIX and Windows:
```

Reply:
Yeah, I'm willing to do this.